src/PKSim.Core/Services/ExpressionProfileUpdater.cs

```diff
@@ -78,24 +80,26 @@ public void SynchroniseSimulationSubjectWithExpressionProfile(ISimulationSubject
       {
          _lazyLoadTask.Load(simulationSubject);
          var moleculeInIndividual = simulationSubject.MoleculeByName(expressionProfile.MoleculeName);
-         synchronizeExpressionProfiles(expressionProfile.Molecule, expressionProfile.Individual, moleculeInIndividual, simulationSubject);
+
+         // ExpressionProfile => SimulationSubject, we want to make sure that the parameters in simulation subject are linked to their expression profile origin parameters
+         synchronizeExpressionProfiles(expressionProfile.Molecule, expressionProfile.Individual, moleculeInIndividual, simulationSubject, updateParameterOriginId:true);
```

Comment:
Here I am updating the parameter origin id so that the parameter in individual knows what parameter in expression profile it's coming from